src/test/compile-fail-fulldeps/auxiliary/unstable-macros.rs

```diff
@@ -0,0 +1,16 @@
+// Copyright 2018 The Rust Project Developers. See the COPYRIGHT
```

Comment:
`compile-fail` (instead of `compile-fail-fulldeps`) would be enough since we don't use procedural macros here.